src/test/java/com/suse/saltstack/netapi/client/SaltStackClientTest.java

```diff
@@ -193,4 +196,4 @@ public void testRunResultAsync() throws SaltStackException {
         assertTrue(retvals.containsKey("minion-1"));
         assertEquals(retvals.get("minion-1"), true);
     }
-}
+}
```

Comment:
@gborp i don't think there is an official profile but some contributors use Eclipse so maybe you can get a profile from them. There is also the style guide section in the readme which covers some formatting rules.